src/snapred/backend/recipe/algorithm/AlignAndFocusReductionAlgorithm.py

```diff
@@ -11,12 +12,14 @@ class AlignAndFocusReductionAlgorithm(PythonAlgorithm):
     def PyInit(self):
         # declare properties
         self.declareProperty("ReductionIngredients", defaultValue="", direction=Direction.Input)
+        self.declareProperty("PixelGroupingParameters", defaultValue="", direction=Direction.Input)
```

Comment:
Shouldn't these be added into the ingredients?

Reply:
When I asked Michael about it initially, he said to add it as an additional input property. I would be fine with moving it into the ingredients, but I don't think that everything that uses ReductionIngredients will need also need access to PixelGroupingParameters

Reply:
From below, the ReductionAlgorithm needs pixel grouping parameters, so I think it should be included in the ingredients.  I'm not sure what Michael's goal was in including it as a separate property.